pkgs/tools/text/sgrep/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchurl, runCommand, lib, m4, makeWrapper }:
+let
+  sgrep = stdenv.mkDerivation rec {
```

Comment:
`let` isn't needed here, just start with `stdenv.mkDerivation`.

Reply:
We need a reference to the derivation output for `passthru.tests.smokeTest`. I figure one alternative would be:

```nix
passthru.tests.smokeTest = callPackage ({ sgrep }: runCommand "test-sgrep" { } ''
…
'') { };
```

… but this looked slightly clumsier than a let. Any recommendrations?

Reply:
Never mind, we can just add sgrep to the top-level callPackage arguments…